fixtures/reconciler/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "react-fixtures",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.0.0-",
+    "react-reconciler": "file:../../build/packages/react-reconciler"
```

Comment:
Missing `prop-types`, `fbjs`, and `object-assign` dependencies?

Reply:
🤷‍♂️ Everything worked okay for me so far. Idk if I even need those in the `packages/react-reconciler` area. There’s also a `rollup` warning right now for this package and apparently I broke bundling `ReactDOM`.

Reply:
Whoops. I thought I was looking at `packages/react-reconciler/package.json` instead of the one in fixtures. Disregard. 😄